fixtures/fixtures.js

```diff
@@ -55,12 +55,14 @@ var Fixtures = {
         ]
     },
     voucherStatus: {
-        "success":
-            {"success":["RVNT12345678"], "fail":[], "invalid":[]},
-        "fail":
-            {"success":[], "fail":["FAL11111111"], "invalid":[]},
-        "invalid":
-            {"success":[], "fail":[], "invalid":["INV1"]},
+        "success": [{"message": 'Voucher is valid'}],
+        "warning": [
+            {"warning": "You have already submitted voucher code :code."},
```

Comment:
Could having an arbitrary code like "RVNT12345678" potentially be confusing? I prefer the idea of having a placeholder - it would be nice to echo the number as Charlie suggested.